### DISCUSSION TITLE
Tropical pacific mask files on MPAS-O Mesh?

### DISCUSSION THREAD

**Discussion by jacob-stu-allen** [E3SM-Project/E3SM#5882] [E3SM model help]
Hello!

I'm wondering if there are some mask templates in netcdf or mask generation scripts for something like the CESM2 pacemaker mask (https://www2.cesm.ucar.edu/working_groups/CVC/simulations/cesm2-pacific_pacemaker_instr.html). 

Thanks for your help,
Jacob

**Comment by xylar** [comment]
We have a python tool in a conda package called `mpas_tools` that creates masks on and MPAS-Ocean mesh, given a geojson file that defines the boundary of the mask and an MPAS-Ocean mesh file.  See:
http://mpas-dev.github.io/MPAS-Tools/stable/mesh_conversion.html#mask-creation-with-python-multiprocessing

You would need to figure out how to create the geojson file you need, but that's typically not too challenging. Here are some example geojson files we use (go into each region's directory and take a look at the `region.geojson` file):
https://github.com/MPAS-Dev/geometric_features/tree/main/geometric_data/ocean/region

**Reply by jacob-stu-allen** [reply]
Amazing, thank you very much Xylar - this is exactly what I needed!

**Reply by xylar** [reply]
No problem. Glad I could help.
